norway_test.go

```diff
@@ -0,0 +1,130 @@
+package meteo_test
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/qba73/meteo"
+)
+
+func TestCreateNewMeteoClient(t *testing.T) {
+	t.Parallel()
+	var c *meteo.NorwayClient
+	resolver, err := meteo.NewWikipediaClient("User")
```

Comment:
```suggestion
	resolver, err := meteo.NewWikipediaClient("DummyUsername")
```
...makes it clearer that this won't actually be used for anything.